sceptre/cli/update.py

```diff
@@ -52,31 +52,46 @@ def update_command(ctx, path, change_set, verbose, yes):
     plan = SceptrePlan(context)
 
     if change_set:
+        delete_change_set = True
+
         change_set_name = "-".join(["change-set", uuid1().hex])
         plan.create_change_set(change_set_name)
         try:
             # Wait for change set to be created
             statuses = plan.wait_for_cs_completion(change_set_name)
             # Exit if change set fails to create
+            atleast_one_ready = False
             for status in list(statuses.values()):
-                if status != StackChangeSetStatus.READY:
+                if status != StackChangeSetStatus.READY \
+                        and status != StackChangeSetStatus.NO_CHANGES:
+                    # TODO: Print an informative message to the user.
```

Comment:
This TODO should be implemented, I think before this gets merged.

Reply:
Fixed in #1192